src/pretix/api/serializers/event.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+
+from pretix.base.models import Event
+
+
+class EventSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Event
+        fields = ('name', 'slug', 'live', 'currency', 'date_from',
```

Comment:
`organizer` is not included, is that intentional? Since you don't provide `id` in `Event` and `Organizer`, you'll probably want either a nested serializer or a `SlugRelatedField` here (and on every other model referencing `Event`, too).